components/authz-service/server/v2/policy.go

```diff
@@ -579,7 +579,7 @@ func (s *policyServer) MigrateToV2(ctx context.Context,
 
 	// Added for testing only; these are handled by data migrations.
 	for _, project := range storage.DefaultProjects() {
-		if _, err := s.store.CreateProject(ctx, &project); err != nil {
+		if _, err := s.store.CreateProject(ctx, &project, false); err != nil {
```

Comment:
Needed to adjust the function signature to accept a 3rd parameter: a boolean indicating to add supporting policies for the project. These are default projects, which should not create policies, hence the `false`.